server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/remote/delivery/DeliveryRunnable.java

```diff
@@ -81,7 +82,7 @@ public DeliveryRunnable(MailQueue queue, RemoteDeliveryConfiguration configurati
     }
 
     public void start() {
-        Scheduler remoteDeliveryScheduler = Schedulers.newBoundedElastic(Schedulers.DEFAULT_BOUNDED_ELASTIC_SIZE, Schedulers.DEFAULT_BOUNDED_ELASTIC_QUEUESIZE, "RemoteDelivery");
+        remoteDeliveryScheduler = Schedulers.newBoundedElastic(Schedulers.DEFAULT_BOUNDED_ELASTIC_SIZE, Schedulers.DEFAULT_BOUNDED_ELASTIC_QUEUESIZE, "RemoteDelivery");
```

Comment:
We should not allow to call start several times without having call the dispose method.
Maybe by adding an `AtomicBoolean` indicating if the Runnable is started or not ?

Reply:
I agree that it could be an improvement however I'm convinced that the sooner we merge this PR the better for our productivity.
Let's keep that enhancement for a later PR.